### PR TITLE
RC: Clarify that monthly network cap applies over all databases in the plan

### PR DESCRIPTION
**Pull request by cmilesb**
DOC-4613

### CODE REVIEW

File: content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md [hunk 1]
```diff
@@ -30,13 +30,15 @@ These plans are currently offered for all new and upgraded Essentials subscripti
...
+| **Monthly<br/> total network<br/> bandwidth<sup>[2](#table-note-2)</sup>** | 5&nbsp;GB | 100&nbsp;GB | 200&nbsp;GB | 400&nbsp;GB | 800&nbsp;GB | 2000&nbsp;GB |
+| **Maximum<br/> throughput<sup>[3](#table-note-3)</sup>** | 100&nbsp;ops/sec | 1000&nbsp;ops/sec | 2000&nbsp;ops/sec | 4000&nbsp;ops/sec | 8000&nbsp;ops/sec | 16000&nbsp;ops/sec |
 {{</table-scrollable>}}
 
 1. <a name="table-note-1" style="display: block; height: 80px; margin-top: -80px;"></a> Database size includes replication where applicable. See [High availability cost impact]({{< relref "/operate/rc/databases/configuration/high-availability#dataset-size" >}}) for more information.
 
-2. <a name="table-note-2" style="display: block; height: 80px; margin-top: -80px;"></a> Assumes request size of 1 KiB. Maximums are capped by actual MB/s reached. To find the MB/s limit, divide the Maximum throughput by 1024.
+2. <a name="table-note-2" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies over all databases in the plan.
```
Comment (dwdougherty) [inline content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md:39]:
```suggestion
2. <a name="table-note-2" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to all databases in the plan.
```

File: content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md [hunk 2]
```diff
@@ -47,9 +49,11 @@ These plans were available before November 2023.
...
-| **Maximum<br/> throughput**                                                       | 100&nbsp;ops/sec | 500&nbsp;ops/sec | 1000&nbsp;ops/sec | 1500&nbsp;ops/sec | 2000&nbsp;ops/sec | 4000&nbsp;ops/sec | 8000&nbsp;ops/sec | 16000&nbsp;ops/sec |
+| **Maximum<br/> throughput<sup>[3](#table-note-3-legacy)</sup>**                                                       | 100&nbsp;ops/sec | 500&nbsp;ops/sec | 1000&nbsp;ops/sec | 1500&nbsp;ops/sec | 2000&nbsp;ops/sec | 4000&nbsp;ops/sec | 8000&nbsp;ops/sec | 16000&nbsp;ops/sec |
 {{</table-scrollable>}}
 
 1. <a name="table-note-1-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> Database size includes replication. See [High availability cost impact]({{< relref "/operate/rc/databases/configuration/high-availability#dataset-size" >}}) for more information.
 
+2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies over all databases in the plan.
```
Comment (dwdougherty) [inline content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md:57]:
```suggestion
2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to all databases in the plan.
```

**Review by dwdougherty — APPROVED** [review]
A couple of suggestions. Approving anyway.

File: content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md (hunk 1 above)
Comment (dwdougherty) [reply content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md:39]: I like it!

File: content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md (hunk 2 above)
Comment (cmilesb) [reply content/operate/rc/subscriptions/view-essentials-subscription/essentials-plan-details.md:57]:
```suggestion
2. <a name="table-note-2-legacy" style="display: block; height: 80px; margin-top: -80px;"></a> The monthly total network bandwidth limit applies to the entire plan. All databases in the plan share the allocated bandwidth limit.
```

**Pull request merged**